styles.css

```diff
@@ -0,0 +1,119 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap');
+
+html {
+  display: block;
+}
+
+body {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+.header {
+  background: #3c3a39;
+  display: flex;
+}
+
+.nav-bar {
+  height: 100%;
+  width: 100%;
+  display: flex;
+  align-items: baseline;
+}
+
+.name a {
+  text-decoration: none;
+  padding: 10px;
+  width: 100%;
+  height: 100%;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  display: flex;
+  color: #fff;
+}
+
+.name {
+  width: 100%;
+  height: 100%;
+}
+
+.menu-icon {
+  padding: 10px;
+  height: 100%;
+  width: 100%;
+  display: flex;
+  justify-content: flex-end;
+}
+
+.greetings {
+  padding-right: 24px;
+  padding-left: 24px;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 56px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+  width: 100%;
+}
+
+.my-job-title {
+  margin-bottom: 3px;
+  padding-right: 24px;
+  padding-left: 24px;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+  display: flex;
+  align-items: center;
+  color: #fff5e1;
+  width: 100%;
+}
+
+.my-job-description {
+  padding-right: 45px;
+  padding-left: 24px;
+  margin-bottom: 15px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 20px;
+  color: #fff;
+  width: 100%;
+  height: fit-content;
+}
+
+.container {
+  width: 100%;
+  height: 590px;
+  background: #1c1a19;
+  background-image: url(images/Illustration.svg);
+  background-repeat: no-repeat;
+  background-position: right bottom -45px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.social-icon-links ul {
+  max-width: 250px;
+  padding-left: 24px;
+  display: flex;
+  justify-content: space-between;
+}
```

Comment:
[Optional]

- [ ] Please consider removing the list default styling (the bullet points) to match the Figma design 💯 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/185491296-770fe04f-be12-4c42-9f8f-e2dcbe11e8f0.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/185491337-d065b3d9-fb38-41d9-8cfe-369335ddddff.png)